src/tir/transforms/default_gpu_schedule.cc

```diff
@@ -113,8 +113,7 @@ bool IsScheduledOnGPU(const BaseFunc& func) {
 
   if (target.defined()) {
     int dev_type = target->GetTargetDeviceType();
-    if (!(dev_type == kDLCUDA || dev_type == kDLMetal || dev_type == kDLROCM ||
-          dev_type == kDLWebGPU)) {
+    if (dev_type != kDLCUDA) {
```

Comment:
Is this intended change?

Reply:
Yes, this was causing a test failure. I will find the link to the discussion

Reply:
https://github.com/apache/tvm/pull/16634#issuecomment-1979832554